src/App/pages/RecipePage/RecipePage.tsx

```diff
@@ -1,56 +1,26 @@
-import Resource from "@app/components/Resource";
-import { URLmap, API_KEY } from "@config/api";
+import React from "react";
+
+import PageStore from "@store/PageStore";
+import { Meta } from "@store/PageStore/types";
+import { useLocalStore } from "@utils/useLocalStote";
+import { observer } from "mobx-react-lite";
 import { useParams } from "react-router-dom";
 
 import RenderPage from "./components/RenderPage";
 
 const RecipePage = () => {
+  const pageStore = useLocalStore(() => new PageStore());
   const { id } = useParams();
```

Comment:
чтобы не было внизу as
```suggestion
  const { id } = useParams<{ id: string }>();
```